docs/installation/setup-gce.rst

```diff
@@ -0,0 +1,31 @@
+.. Single Source Instructions
+
+============================================
+Setting Up Nodes Using Google Compute Engine
+============================================
+
+.. begin-body
+
+You can get a Flocker cluster running using Google Compute Engine.
+You'll need to setup at least two nodes.
+
+#. Create a new cloud server:
+
+   * Visit the `Google Cloud Console <https://console.cloud.google.com/>`_.
+   * Navigate to the project you want to use.
+   * Use the menu in the upper left to navigate to ``Compute Engine``.
+   * Click ``Create Instance``.
+   * Choose a name and a zone for your instance. Zone must be the same for all instances.
```

Comment:
Ah referencing http://rhodesmill.org/brandon/2012/one-sentence-per-line/

Interesting stuff. Although not the way I would want to solve that problem :)

I'd prefer having semantic `diff` programs. If I win the lottery I'll make one for C++ to show you what I'm talking about. Essentially, in my mind the problem is his diff program is unaware that newlines are not significant in rst files. Instead, he should use a diff program that knows that newlines are not significant. Similarly for something like C++, for some changes I would much rather review the AST diff than the actual code diff.